test/interface/jacobian.jl

```diff
@@ -8,49 +8,180 @@ using Test
     nothing
   end
 
-  function g(J, u, h, p, t)
+  njacs = Ref(0)
+  function jac(J, u, h, p, t)
+    njacs[] += 1
     J[1, 1] = 1 - h(p, t - 1)[1]
     nothing
   end
 
+  nWfacts = Ref(0)
+  function Wfact(W, u, h, p, dtgamma, t)
+    nWfacts[] += 1
+    W[1,1] = dtgamma * (1 - h(p, t - 1)[1]) - 1
+    nothing
+  end
+
+  nWfact_ts = Ref(0)
+  function Wfact_t(W, u, h, p, dtgamma, t)
+    nWfact_ts[] += 1
+    W[1,1] = 1 - h(p, t - 1)[1] - inv(dtgamma)
```

Comment:
```suggestion
    W[1,1] = h(p, t - 1)[1] - inv(dtgamma)
```

IIRC

Reply:
Really? I just checked the implementation in ModellingToolkit, and there it is defined as
W_t = - LinearAlgebra.I/gam + jac in
https://github.com/JuliaDiffEq/ModelingToolkit.jl/blob/fd98d1b7b82882c8abd62f9a57b9945ea035ddd2/src/systems/diffeqs/diffeqsystem.jl#L188. In our case the Jacobian is a 1x1 matrix with element `1 - h(p, t  - 1)[1]`.

Reply:
oh yes, I did the jac wrong